### PR TITLE
added contributing.md file

### PR DESCRIPTION
**Pull request by satyamgupta1495**
Resolves https://github.com/OSSPhilippines/freefolio/issues/16

### CODE REVIEW

**Comment by jofftiquez** [comment]
Good job, thank you for this @satyamgupta1495

**Comment by satyamgupta1495** [comment]
> Good job, thank you for this @satyamgupta1495

Always happy to help! I'd love to contribute more in any other way possible!

**Review by rgvillanueva28 — COMMENTED** [review]
Noticed some minor errors

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+- Take a look at the existing [Issues](https://github.com/OSSPhilippines/freefolio/issues) or [create a new issue](https://github.com/OSSPhilippines/freefolio/issues/new/choose)!
+- [Fork the Repo](https://github.com/OSSPhilippines/freefolio/fork). Then, create a branch for any issue that you are working on. Finally, commit your work.
+- Create a **[Pull Request](https://github.com/OSSPhilippines/freefolio/compare)** (_PR_), which will be promptly reviewed and given suggestions for improvements by the community.
+- Add screenshots or screen captures to your Pull Request to help us understand the effects of the changes proposed in your PR.
+
+---
+
+## ⭐ HOW TO MAKE A PULL REQUEST:
+
+**1.** Start by making a Fork of the [**YourQuotes**](https://github.com/OSSPhilippines/freefolio) repository. Click on the <a href="https://github.com/OSSPhilippines/freefolio/fork"><img src="https://i.imgur.com/G4z1kEe.png" height="21" width="21"></a>Fork symbol at the top right corner.
```
Comment (rgvillanueva28) [inline CONTRIBUTING.md:27]: YourQuotes should be Freefolio

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+**7.** Push your local commits to the remote repository:
+
+```bash
+git push origin YourBranchName
+```
+
+**8.** Create a [Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)!
+
+**9.** **Congratulations!** You've made your first contribution to [**YourQuotes**](https://github.com/OSSPhilippines/freefolio)! 🙌🏼
```
Comment (rgvillanueva28) [inline CONTRIBUTING.md:77]: YourQuotes should be Freefolio

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+**2.** Clone your new fork of the repository in the terminal/CLI on your computer with the following command:
+
+```bash
+git clone https://github.com/<your-github-username>/freefolio
+```
+
+**3.** Navigate to the newly created LinkFree project directory:
+
+```bash
+cd YourQuotes
```
Comment (rgvillanueva28) [inline CONTRIBUTING.md:38]: YourQuotes should be freefolio

**Comment by satyamgupta1495** [comment]
Hey @rgvillanueva28 , Thanks for pointing out the mistake I have solved it, could you plz check it?

**Comment by rgvillanueva28** [comment]
Looks good to me.

**Pull request merged**